centreon/src/Core/TimePeriod/Domain/Model/TimeRange.php

```diff
@@ -42,6 +47,15 @@ class TimeRange implements \Stringable
      */
     public function __construct(string $timeRange)
     {
+
+        $resolvedTimeRangeAlias = $this->resolveTimeRangeAlias($timeRange);
+        $timeRange = $resolvedTimeRangeAlias['value'];
+        if ($resolvedTimeRangeAlias['isFullDayAlias']) {
+            $this->timeRange = $timeRange;
+
+            return;
+        }
```

Comment:
why do you need to return an array from resolveTimeRangeAlias ? you can change the value of $this->timeRAnge directly in resolveTimeRangeAlias without using an array ;)

Reply:
resolved

Reply:
ok